airflow/config_templates/config.yml

```diff
@@ -1477,6 +1477,20 @@
       type: boolean
       example: ~
       default: "True"
+    - name: email_on_dag_import_failure
+      description: |
+        Whether email alerts should be sent when a dag fails to import
+      version_added: ~
+      type: boolean
+      example: ~
+      default: "True"
```

Comment:
We don't need a setting and a list of emails -- empty = don't email.

Reply:
oh yeah, that's fair. I'll make the adjustment